src/util/routes.ts

```diff
@@ -0,0 +1,7 @@
+export const parseRoute = (route: string): { page: string; zna?: string } => {
+	const split = route.split('/');
+	const page = split[0];
+	const zna = split[1];
+
+	return { page, zna };
+};
```

Comment:
Was this copied from `zNS-dapp`? If yes, the ordering of the zNA and app have changed. It's now `zero.live/[zna]/[app]/[page]`.